src/Sarif/Readers/SarifContractResolver.cs

```diff
@@ -34,8 +35,14 @@ protected override JsonContract CreateContract(Type objectType)
             else if (objectType == typeof(NotificationLevel))
                 contract.Converter = EnumConverter.Instance;
 
-            else if (objectType == typeof(AlgorithmKind))
-                contract.Converter = AlgorithmKindConverter.Instance;
+            if (objectType == typeof(AlgorithmKind))
```

Comment:
Why remove the else's? It causes unnecessary tests (minor), and it suggests to the reader that the cases are not exclusive, which they are.